code/lighting/lighting.cpp

```diff
@@ -298,8 +298,12 @@ void light_add_point(const vec3d *pos, float r1, float r2, float intensity, floa
 
 void light_add_point_unique(const vec3d *pos, float r1, float r2, float intensity, float r, float g, float b, int affected_objnum, float spec_r, float spec_g, float spec_b, bool specular)
 {
-	Assert(r1 >0);
-	Assert(r2 >0);
+	Assertion(r1 > 0.0f, "Invalid radius r1 specified for light: %f. Radius must be > 0.0f. Examine stack trace to determine culprit.\n", r1);
+	Assertion(r2 > 0.0f, "Invalid radius r2 specified for light: %f. Radius must be > 0.0f. Examine stack trace to determine culprit.\n", r2);
+
+	if (r1 < 0.0001f || r2 < 0.0001f)
```

Comment:
I copied across this condition because it appeared in two out of the four light functions, and it seemed reasonable to have it in all four.  You think it shouldn't even have been added in the first place?